test/fixtures/dialects/hive/repair_table.sql

```diff
@@ -0,0 +1,12 @@
+-- REPAIR TABLE with all optional syntax
+MSCK REPAIR TABLE table_identifier ADD PARTITIONS;
+MSCK REPAIR TABLE table_identifier DROP PARTITIONS;
+MSCK REPAIR TABLE table_identifier SYNC PARTITIONS;
+
+-- REPAIR TABLE with no optional syntax
+MSCK REPAIR TABLE table_identifier;
+
+-- run MSCK REPAIR TABLE to recovers all the partitions
+MSCK REPAIR TABLE t1;
+
+MSCK REPAIR TABLE emp_part DROP PARTITIONS;
```

Comment:
Add some tests cases where the REPAIR keyword isn't there

Reply:
Added `test/fixtures/dialects/five/msck_table.sql`